### PR TITLE
Added page about adapter translations including Weblate

### PR DESCRIPTION
**Pull request by UncleSamSwiss**

### CODE REVIEW

File: docs/en/dev/adaptertranslate.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Automated translations are often not good enough or confusing, therefore ioBroker offers the Weblate platform for managed community translations:
+
+https://weblate.iobroker.net/
+
+In Weblate members of the community can easily manage translations to any number of languages for all included ioBroker adapters.
+
+To add your adapter to Weblate, please follow [these guidlines](https://github.com/ioBrokerTranslator/doc/blob/master/README.md).
+
+Weblate currently only manages strings in the admin user interface. It does not alter `io-package.json` or do anything to your documentation.
```
Comment (Apollon77) [inline docs/en/dev/adaptertranslate.md:47]: Maybe add that the release news are handled aztomatically by the release scriopt and link to it? ;-)

Comment (UncleSamSwiss) [reply docs/en/dev/adaptertranslate.md:47]: If it makes sense, I would love to. But I have no clue what the release script is. I didn't find anything in the documentation and it doesn't seem to be part of Adapter Creator.

Comment (Apollon77) [reply docs/en/dev/adaptertranslate.md:47]: Ok, I asdd it ... all good. I merge tonight

**Comment by UncleSamSwiss** [comment]
This PR is still open, is there anything I can do?

**Comment by Apollon77** [comment]
No, sorry I lost from radar ... will merge

**Pull request merged**